apps/remix-ide/src/app/files/fileManager.js

```diff
@@ -140,7 +140,7 @@ class FileManager extends Plugin {
   refresh () {
     const provider = this.fileProviderOf('/')
     // emit rootFolderChanged so that File Explorer reloads the file tree
-    provider.event.emit('rootFolderChanged', provider.workspace || '/')
```

Comment:
we are listening on the `rootFolderChanged` of the provider here too: https://github.com/ethereum/remix-project/blob/9238ca9dfdc4649845f1902ba271b2cd2ff36324/apps/solidity-compiler/src/app/compiler-api.ts#L227

might worth keeping it

Reply:
nobody else uses this method but dgit and dgit only works without remixd on